pest/src/prec_climber.rs

```diff
@@ -146,7 +145,7 @@ impl<R: RuleType> PrecClimber<R> {
     /// let primary = |pair| {
     ///     consume(pair, climber)
     /// };
-    /// let infix = |lhs: i32, op: Pair<Rule, StringInput>, rhs: i32| {
+    /// let infix = |lhs: i32, op: Pair<Rule, StrInput>, rhs: i32| {
```

Comment:
This needs to use the new `'i` signature. Best solution would probably be to grep after `Input`.

Reply:
:+1: